docs/topics_5/web-adding-custom-labels.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='web_adding_custom_labels_{context}']
+= Adding custom labels
+
+You can add custom labels to use in each project. When you add a label, it is disabled by default. If you no longer need a label, you can delete it.
+
+.Procedure
+
+. Click *Add label*.
```

Comment:
Where is the user supposed to find something to click? You can't begin a procedure with "Click x". It has to be more specific like this

1. In the MTA web console, click Labels.
2. Click Add label.
3. If you want to upload a labels file, click Browse ....
4. If you want to register a server path...

Do not use text box. IBM says "field". Enter the path of the server-side file or of a directory containing the labels files in the *Labels path* field.